contrib/PKGBUILD

```diff
@@ -0,0 +1,35 @@
+# Maintainer: Bruno Pagani (a.k.a. ArchangeGabriel) <archange@archlinux.org>
+# Contributor: Mirco Tischler <mt-ml at gmx dot de>
+
+pkgname=fwupd
+pkgver=LATEST
```

Comment:
It wasn't clear to me, will the packaging automatically pick up the correct version to swap into this (from `ninja-dist `or from parsing latest git tag)?
If not, consider to use sed to swap in the right version from the shell script (similar to what deb and rpm are doing to parse the version).

The reasoning for getting this right would be if we ever decided to archive the build artifacts at tagged releases (which hasn't been confirmed).